lib/ansible/modules/packaging/language/pip.py

```diff
@@ -271,15 +336,15 @@ def _get_packages(module, pip, chdir):
     return (command, out, err)
 
 
-def _is_present(name, version, installed_pkgs, pkg_command):
+def _is_present(module, req, installed_pkgs, pkg_command):
     '''Return whether or not package is installed.'''
     for pkg in installed_pkgs:
         if '==' in pkg:
             pkg_name, pkg_version = pkg.split('==')
         else:
```

Comment:
Hm... Does it skip `--editable` dists this way? Can/should `pip list` improve this experience?